configuration/eclipseiotmarket.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: documentation
+title: Eclipse IoT Marketplace
+---
+
+{% include base.html %}
+
+# Eclipse IoT Marketplace Integration
+
+The [Eclipse IoT Marketplace](https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title) hosts third-party add-ons that can be browsed and installed to openHAB through this integration.
+
+## Installation
+
+Install the market integration from Add-ons → Misc → Eclipse IoT Market.
+
+![iot marketplae addon](images/eclipseiotmarket_addons.png)
+
+## Usage
+
+When you now refresh your browser, you will see new entries in the "Bindings" tab of the Add-ons menu. As the new entries come with logos and descriptions, the card layout (instead of the list layout, can be selected in the top right) is best for browsing the entries. To filter the view to show only third-party add-ons from the marketplace, you can simply type "market" in the search bar:
```

Comment:
If somebody already installed the add-on and wants to know how to use it, the first sentence here is a bit strange. How about:

> Whit the Eclipse IoT Marketplace installed, you will see additional entries in the "Bindings" tab of the Add-ons menu.